src/absil/il.fsi

```diff
@@ -1815,6 +1818,7 @@ val mkILCustomAttrsFromArray: ILAttribute[] -> ILAttributes
 val storeILCustomAttrs: ILAttributes -> ILAttributesStored
 val mkILCustomAttrsReader: (int32 -> ILAttribute[]) -> ILAttributesStored
 val emptyILCustomAttrs: ILAttributes
+val emptyILCustomAttrsStored: ILAttributesStored
```

Comment:
Might be good to put all these smaller changes in a separate PR

Reply:
Yea that makes sense. Less changes outside `ilread.fs` the better.